services/app-api/src/testHelpers/launchDarklyHelpers.ts

```diff
@@ -7,7 +7,7 @@ import {
 import { defaultFeatureFlags } from '../launchDarkly/launchDarkly'
 
 function testLDService(mockFeatureFlags?: FeatureFlagSettings): LDService {
-    const featureFlags = defaultFeatureFlags
+    const featureFlags = Object.assign({}, defaultFeatureFlags)
```

Comment:
We are using that `defaultFeatureFlags` object to derive types. I don't know how much of a refactor it would take if we change it to a function and try to get types from it. So I opted to copy it instead. I would like to split it into a new ticket if turning it into a function is the way we should go though.

Reply:
def don't need to change this, just seeing you solve the same problem we saw elsewhere